### PR TITLE
ext/audio: update audiovm qubesdb entry

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: qubes/ext/audio.py [hunk 1]
```diff
@@ -66,15 +75,11 @@ def on_property_set(self, subject, event, name, newvalue, oldvalue=None):
         if newvalue:
             audiovm = "audiovm-" + newvalue.name
             subject.tags.add(audiovm)
+            self.set_qubesdb_audiovm(subject)
```
Comment (marmarek) [inline qubes/ext/audio.py:78]: This should be under `if subject.is_running()`

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by marmarek** [comment]
oh, and also a lot of tests failed

**Comment by fepitre** [comment]
PipelineRetry

**Comment by fepitre** [comment]
PipelineRetry

**Comment by fepitre** [comment]
PipelineRetry

**Comment by fepitre** [comment]
PipelineRetry

File: qubes/ext/audio.py [hunk 2]
```diff
@@ -20,69 +20,96 @@
...
 import qubes.ext
 
+
 class AUDIO(qubes.ext.Extension):
     # pylint: disable=unused-argument
     @staticmethod
     def attached_vms(vm):
         for domain in vm.app.domains:
-            if getattr(domain, 'audiovm', None) and domain.audiovm == vm:
+            if getattr(domain, "audiovm", None) and domain.audiovm == vm:
```
Comment (DemiMarie) [inline qubes/ext/audio.py:29]:
```suggestion
            if getattr(domain, "audiovm", None) == vm:
```

should be equivalent

File: qubes/ext/audio.py [hunk 3]
```diff
@@ -20,69 +20,96 @@
...
+        # Ensure that qube is ready
+        if not vm.untrusted_qdb:
+            return
+
+        # Add AudioVM Xen ID for gui-agent
+        if getattr(vm, "audiovm", None):
+            if vm != vm.audiovm and vm.audiovm.is_running():
+                vm.untrusted_qdb.write(
+                    "/qubes-audio-domain-xid", str(vm.audiovm.xid)
+                )
```
Comment (DemiMarie) [inline qubes/ext/audio.py:43]:
```suggestion
        audiovm = getattr(vm, "audiovm", None):
        if audiovm is not None:
            if audiovm != vm and audiovm.is_running():
                vm.untrusted_qdb.write(
                    "/qubes-audio-domain-xid", str(audiovm.xid)
                )
```

**Review by DemiMarie — COMMENTED** [review]
Nits

**Review by marmarek — APPROVED** [review]

**Pull request merged**